### PR TITLE
Build System Updates / Code Cleanup

### PR DESCRIPTION
**Pull request by mattleibow**
Updated the repository
 - for a more modern build system:
    - AppVeyor file
    - using Cake build
 - added NuGet
 - using WP8 instead of WP7
 - Android sample now uses appcompat for cool looks
 - improvements to the iOS designer support

### CODE REVIEW

**Comment by mattleibow** [comment]
No more updates, reverted for a later stage

File: nuget/Xamarin.Controls.SignaturePad.nuspec [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    <projectUrl>https://github.com/xamarin/SignaturePad</projectUrl>
+    <iconUrl>http://ICON_URL_HERE_OR_DELETE_THIS_LINE</iconUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>Makes capturing, saving, and displaying signatures extremely simple.</description>
+    <copyright>Copyright (c) Xamarin Inc. 2016</copyright>
+    <tags>xamarin,signature,handwriting,windows,ios,android,windows phone</tags>
+  </metadata>
+  <files>
+    <file src="output/android/SignaturePad.dll" target="lib\MonoAndroid\SignaturePad.dll" />
+    <file src="output/ios/SignaturePad.dll" target="lib\MonoTouch\SignaturePad.dll" />
```
Comment (bholmes) [inline nuget/Xamarin.Controls.SignaturePad.nuspec]: Drop Classic support.  Take it out of the build too

File: samples/Sample.iOS/Sample.iOS.csproj [hunk 2]
```diff
@@ -124,12 +152,15 @@
   <ItemGroup>
     <None Include="Info.plist" />
     <None Include="Entitlements.plist" />
+    <InterfaceDefinition Include="MainStoryboard.storyboard" />
```
Comment (bholmes) [inline samples/Sample.iOS/Sample.iOS.csproj]: Consider adding 2 samples.  One with the view added in the storyboard.  One with the view added in code.  Maybe one app with 2 buttons on main screen and a nav controller.

File: src/SignaturePad.iOS/SignaturePadView.cs [hunk 3]
```diff
@@ -161,12 +233,12 @@ public SignaturePadView ()
 
 		public SignaturePadView (NSCoder coder) : base (coder)
 		{
-			Initialize ();
+            Initialize(/* ? baseProperties: false ? */);
```
Comment (bholmes) [inline src/SignaturePad.iOS/SignaturePadView.cs]: whitespace error

File: src/SignaturePad.iOS/SignaturePadView.cs [hunk 4]
```diff
@@ -175,9 +247,11 @@ public SignaturePadView (CGRect frame)
 			Initialize ();
 		}
 
-		void Initialize ()
+		void Initialize (bool baseProperties = true)
 		{
-			BackgroundColor = UIColor.FromRGB(225, 225, 225);
+            if (baseProperties) {
```
Comment (bholmes) [inline src/SignaturePad.iOS/SignaturePadView.cs]: whitespace error

**Comment by mattleibow** [comment]
Closing this one as this is now going to be merged with #31

**Pull request closed without merging**